.github/workflows/pr.yaml

```diff
@@ -0,0 +1,35 @@
+name: pr
+
+on:
+  push:
+    branches:
+      - "pull-request/[0-9]+"
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}-${{ github.event_name }}
+  cancel-in-progress: true
+
+# TODO: I would love to not need RAPIDS shared workflows for these builds, but
+# for getting things stood up quickly that's the fastest route.
+jobs:
+  pr-builder:
+    needs:
+      - wheel-build-ucx
+      #- wheel-tests-ucx
```

Comment:
Why is this commented out? Is there something currently failing?

Reply:
No reason, just a holdover from an earlier iteration. I'll uncomment this.